servers/cromwell/jobs/test/test_jobs_controller.py

```diff
@@ -539,7 +539,7 @@ def _request_callback(request, context):
                 'stdout': std_out,
                 'taskName': 'analysis',
                 'timestamp': response_timestamp
-             },{
+            },{
```

Comment:
unrelated?

Reply:
unrelated, but it's a formatting fix I spotted while working on this